build/Nuget-prep.ps1

```diff
@@ -0,0 +1,36 @@
+# Copy files needed for Nuget package generation to appropriate folder
```

Comment:
Consider renaming the file to reflect what it actually does. "Nuget-prep" make me think it gets ready for some as aspect of something to do with nuget - not clear what though. And it actually seems to pack.

Reply:
Yeah I caught this as well. Renaming the script to Nuget-Pack and adding comments in my upcoming commit.